WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -1515,7 +1515,13 @@ private extension OrderDetailsDataSource {
     }
 
     func hasCardPresentEligiblePaymentGatewayAccount() -> Bool {
-        resultsControllers.paymentGatewayAccounts.contains(where: \.isCardPresentEligible)
+        let accounts = resultsControllers.paymentGatewayAccounts
```

Comment:
Hide the Collect Payment button if there are more than one payment gateways in the system